README.md

```diff
@@ -34,14 +34,24 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. If you were to describe semantic HTML to the next cohort of students, what would you say?
 
+semantic explains what it does for people reading your code / organizes the project. not meant for aesthetics.
+
 2. Name two big differences between ```display: block;``` and ```display: inline;```.
 
+block becomes a box module, inline is neutral / next to elements around it.
+
 3. What are the 4 areas of the box model?
 
+margin border padding content
+
 4. While using flexbox, what axis does the following property work on: ```align-items: center```?
 
+main/horizontal axis
+
```

Comment:
For this one it is along the cross axis, where they (vertical axis) meets the x (horizontal axis). It takes all the children of that element and aligns them at that center point.  
y
|
item] --x-axis-->